### PR TITLE
Feat/web ca constants

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/web-ca`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-5098, MANAGER-5045, MANAGER-5051, MANAGER-5074, MANAGER-5013, MANAGER-5090
| License          | BSD 3-Clause

## Description

- Remove unused constants
- Add guides for Web in CA 
- Add order URL for CA and refactor to use only one source for order URLS if possible and relevant

### CODE REVIEW

File: packages/manager/apps/web/client/app/domain/webhosting-enable/domain-enable-web-hosting.controller.js [hunk 1]
```diff
@@ -76,9 +78,7 @@ angular.module('App').controller(
         .then(({ modules, offers, user }) => {
           this.model.moduleTemplates = filter(modules, { branch: 'stable' });
           this.offers = compact(offers);
-          this.hostingUrl =
-            this.constants.urls.hosting[user.ovhSubsidiary] ||
-            this.constants.urls.hosting.FR;
+          this.hostingUrl = this.ORDER_URLS.orderHosting[user.ovhSubsidiary];
```
Comment (FredericEspiau) [inline packages/manager/apps/web/client/app/domain/webhosting-enable/domain-enable-web-hosting.controller.js:81]: Sure this isn't required anymore ?

Comment (marie-j) [reply packages/manager/apps/web/client/app/domain/webhosting-enable/domain-enable-web-hosting.controller.js:81]: All subs hosting URLS are defined and we use `ng-if` for some display

File: packages/manager/apps/web/client/app/app.js [hunk 2]
```diff
@@ -30,7 +30,9 @@ import ovhManagerCatalogPrice from '@ovh-ux/manager-catalog-price';
 import ovhManagerNavbar from '@ovh-ux/manager-navbar';
 import ovhManagerProductOffers from '@ovh-ux/manager-product-offers';
 import uiRouter, { RejectType } from '@uirouter/angularjs';
-import ovhManagerServerSidebar from '@ovh-ux/manager-server-sidebar';
+import ovhManagerServerSidebar, {
+  ORDER_URLS,
+} from '@ovh-ux/manager-server-sidebar';
```
Comment (cbourgois) [inline packages/manager/apps/web/client/app/app.js:35]: Thinking: I don't know what is the best design : put all order urls in the same constants files and share it between all packages or have dedicated url in each module and expose them.

File: packages/manager/apps/web/client/app/app.js [hunk 3]
```diff
@@ -129,29 +131,28 @@ angular
...
     ORDER_URL: config.constants.ORDER_URL,
   })
   .constant('LANGUAGES', config.constants.LANGUAGES)
   .constant('website_url', config.constants.website_url)
+  .provider(
+    'ORDER_URLS',
+    /* @ngInject */ (coreConfigProvider) => ({
+      $get: () => ORDER_URLS[coreConfigProvider.getRegion()],
+    }),
+  )
```
Comment (cbourgois) [inline packages/manager/apps/web/client/app/app.js:155]: Could be in `@ovh-ux/manager-server-sidebar` ? (to be available for all)

File: packages/manager/apps/web/client/app/app.js (hunk 2 above)
Comment (marie-j) [reply packages/manager/apps/web/client/app/app.js:35]: @cbourgois I think the current issue is that modules are not all clearly defined (for example it's not possible to import stuff from hosting module without having to install web manager)

File: packages/manager/apps/web/client/app/app.js (hunk 3 above)
Comment (marie-j) [reply packages/manager/apps/web/client/app/app.js:155]: I'll have a look to take care of this :)

**Pull request merged**